src/components/App/App.component.jsx

```diff
@@ -21,10 +19,7 @@ function App() {
             <Private exact path="/favorites">
               <FavoritesPage />
             </Private>
-            <Route exact path="/404">
```

Comment:
If you we use a route like "videoDetail/:id" instead. That way, we are being more specifics that we expect the id of the video for this page, other wise any other route not specified here will be treated as videoID.
If we don't specify the path for the NotFound component route, it will be the destination to any not specified routes  (not valid routes).